bin/downgrade-select

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+exec fzf \
+  --tac \
+  --border \
+  --header-lines 1 \
+  --tiebreak=begin \
+  --select-1 \
+  --exit-0
```

Comment:
We don't need our current `== 0` and `== 1` branches on the list of candidates, if we used these options.

Reply:
Good catch!